contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -130,6 +134,8 @@ The first part of the token is the `token-id`.  The second part is the `token-se
 
 This new type of token is different from the current CSV token authenticator that is currently part of Kubernetes.  The CSV token authenticator requires an update on disk and a restart of the API server to update/delete tokens.  As we prove out this token mechanism we may wish to deprecate and eventually remove that mechanism.
```

Comment:
I agree that we should make tokens dynamic.  But I am unclear why bootstrap tokens are different from "normal" auth tokens?